TeacherWorkout.Data/Repositories/LessonRepository.cs

```diff
@@ -32,6 +32,11 @@ public PaginatedResult<Lesson> PaginatedList(LessonFilter filter)
                 result = result.Where(l => l.Theme.Id == filter.ThemeId);
             }
 
+            if (!string.IsNullOrEmpty(filter.Term))
+            {
+                result = result.Where(l => l.Title.Contains(filter.Term) || l.Description.Contains(filter.Term));
```

Comment:
Consider ignoring case and accents (e.g. diacritice): `EF.Functions.ILike(l.Title, $"%{filter.Term}%"))`

Reply:
Thanks for the remarks. I will update the code. For the accent case, I noticed that EF has a function `EF.Functions.Unaccent()` but only works with UTF-8 dbs, I'll check that and treat accents also.